tests/integration/clients/test_async_websocket_client.py

```diff
@@ -0,0 +1,91 @@
+"""Memory usage of Websocket clients."""
```

Comment:
This file should be called `test_websocket_client` and the description should be more general, as it may be used for other tests in the future.